src/main/java/org/pac4j/dropwizard/J2EHelper.java

```diff
@@ -0,0 +1,161 @@
+package org.pac4j.dropwizard;
+
+import java.lang.reflect.InvocationTargetException;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.servlet.DispatcherType;
+import javax.servlet.Filter;
+import javax.servlet.FilterRegistration;
+
+import org.pac4j.core.config.Config;
+import org.pac4j.core.config.ConfigFactory;
+import org.pac4j.core.config.ConfigSingleton;
+import org.pac4j.core.context.Pac4jConstants;
+import org.pac4j.dropwizard.Pac4jFactory.ServletCallbackFilterConfiguration;
+import org.pac4j.dropwizard.Pac4jFactory.ServletLogoutFilterConfiguration;
+import org.pac4j.dropwizard.Pac4jFactory.ServletSecurityFilterConfiguration;
+import org.pac4j.j2e.filter.ApplicationLogoutFilter;
+import org.pac4j.j2e.filter.CallbackFilter;
+import org.pac4j.j2e.filter.SecurityFilter;
+
+import io.dropwizard.setup.Environment;
+import javassist.CannotCompileException;
+import javassist.ClassPool;
+import javassist.CtClass;
+import javassist.CtField;
+import javassist.CtNewMethod;
+import javassist.NotFoundException;
+
+/**
+ * 
+ * @author Evan Meagher
+ * @author Victor Noel - Linagora
+ * @since 1.0.0
+ *
+ */
+public class J2EHelper {
```

Comment:
I meant to make the class `final`: `public final class J2EHelper`.
Not very useful indeed as the `private` constructor prevents subclassing, but it makes things more explicit.
Not a big deal anyway.